File: regexlib/python_re_test_file/regexlib_7906.py
# 7906
# &lt;script[\s\S]*?&lt;/script([\s\S]*?)&gt;
# EXPONENT
# nums:4
# EXPONENT AttackString:""+"&lt;script&lt;/script"*256+"! _1SLQ_2"

import re
from time import perf_counter

regex = """&lt;script[\s\S]*?&lt;/script([\s\S]*?)&gt;"""
REGEX = re.compile(regex)
for i in range(0, 150000):
    ATTACK = "" + "&lt;script&lt;/script" * i * 1 + "! _1SLQ_2"
    LEN = len(ATTACK)
    BEGIN = perf_counter()
    m = REGEX.search(ATTACK)
    # m = REGEX.match(ATTACK)
    DURATION = perf_counter() - BEGIN
    print(f"{i *1}: took {DURATION} seconds!")

File: regexlib/javascript_test_file/regexlib_7906.js
/* 7906
 * &lt;script[\s\S]*?&lt;/script([\s\S]*?)&gt;
 * EXPONENT
 * nums:3
 * EXPONENT AttackString:""+"&lt;script&lt;/script"*256+"! _1SLQ_2"
 */
var REGEX = new RegExp("&lt;script[\s\S]*?&lt;/script([\s\S]*?)&gt;")
for(var i = 1; i <= 5000000; i++) {
    var time = Date.now();
    var attack_str = '' + '&lt;script&lt;/script'.repeat(i*1) + '! _1SLQ_2'
    REGEX.exec(attack_str)
    // REGEX.test(attack_str);
    // attack_str.search(REGEX)
    // attack_str.match(REGEX)
    var time_cost = Date.now() - time;
    console.log(i * 1 + ": " + time_cost+" ms")
}